MapboxNavigation/NavigationView.swift

```diff
@@ -75,6 +75,34 @@ open class NavigationView: UIView {
     lazy var muteButton = FloatingButton.rounded(image: Images.volumeUp, selectedImage: Images.volumeOff)
     lazy var reportButton = FloatingButton.rounded(image: Images.feedback)
     
+    public enum FloatingButtonsPositions {
+        case topRight, topLeft
```

Comment:
This PR uses `leading` and `trailing` constraints, which means the sides are swapped in right-to-left languages. To avoid confusion, consider calling the cases `topLeading` and `topTrailing`, which would be consistent with `NSRectAlignment`. (Ideally we’d just use `NSRectAlignment`, but that enumeration is only available on iOS 13 and above.)